src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -164,7 +164,7 @@ const PriceTextInput = ({
             ? 'genieBlue'
             : listPrice != null
             ? 'textSecondary'
-            : 'gray700'
+            : 'blue400'
```

Comment:
I believe this color is supposed to go back to gray when a price is set but confirm with product